server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSInquireSPOptionsCheckUtility.java

```diff
@@ -0,0 +1,1880 @@
+        checkBrowseMutuallyExclusive(tdqueueContext);
+        if (!tdqueueContext.START().isEmpty() || !tdqueueContext.END().isEmpty()) {
+          checkBrowsingInvalidOptions(tdqueueContext, CICSParser.TDQUEUE);
+          checkBrowsingHasNotParameter(tdqueueContext, CICSParser.TDQUEUE);
+        } else checkStatementHasParameter(tdqueueContext, CICSParser.TDQUEUE);
+        break;
+
+      case CICSParser.RULE_cics_inquire_terminal:
+        CICSParser.Cics_inquire_terminalContext terminalContext =
+            (CICSParser.Cics_inquire_terminalContext) ctx;
+        checkBrowseMutuallyExclusive(terminalContext);
+        if (!terminalContext.START().isEmpty() || !terminalContext.END().isEmpty()) {
+          checkBrowsingInvalidOptions(terminalContext, CICSParser.TERMINAL, CICSParser.NETNAME);
+          checkBrowsingHasNotParameter(terminalContext, CICSParser.TERMINAL);
+          checkStatementHasParameter(terminalContext, CICSParser.NETNAME);
+        } else checkStatementHasParameter(terminalContext, CICSParser.TERMINAL);
+        break;
+
+      case CICSParser.RULE_cics_inquire_trandumpcode:
+        CICSParser.Cics_inquire_trandumpcodeContext trandumpcodeContext =
+            (CICSParser.Cics_inquire_trandumpcodeContext) ctx;
+        checkBrowseMutuallyExclusive(trandumpcodeContext);
+        if (!trandumpcodeContext.START().isEmpty() || !trandumpcodeContext.END().isEmpty()) {
+          checkBrowsingInvalidOptions(trandumpcodeContext, CICSParser.TRANDUMPCODE);
+          checkBrowsingHasNotParameter(trandumpcodeContext, CICSParser.TRANDUMPCODE);
+        } else checkStatementHasParameter(trandumpcodeContext, CICSParser.TRANDUMPCODE);
+        break;
+
+      case CICSParser.RULE_cics_inquire_tranclass:
+        CICSParser.Cics_inquire_tranclassContext tranclassContext =
+            (CICSParser.Cics_inquire_tranclassContext) ctx;
+        checkBrowseMutuallyExclusive(tranclassContext);
+        if (!tranclassContext.START().isEmpty() || !tranclassContext.END().isEmpty()) {
+          if (!tranclassContext.END().isEmpty())
+            checkHasIllegalOptions(tranclassContext.AT(), "AT with END");
+          checkBrowsingInvalidOptions(tranclassContext, CICSParser.TRANCLASS);
+          checkBrowsingHasNotParameter(tranclassContext, CICSParser.TRANCLASS);
+        } else checkStatementHasParameter(tranclassContext, CICSParser.TRANCLASS);
+
+        if (!tranclassContext.NEXT().isEmpty())
+          checkHasIllegalOptions(tranclassContext.AT(), "AT with NEXT");
+
+        break;
+
+      case CICSParser.RULE_cics_inquire_transaction:
+        CICSParser.Cics_inquire_transactionContext transactionContext =
+            (CICSParser.Cics_inquire_transactionContext) ctx;
+        checkHasMutuallyExclusiveOptions(
+            "START or AT or END or NEXT",
+            transactionContext.START(),
+            transactionContext.AT(),
+            transactionContext.END(),
+            transactionContext.NEXT());
```

Comment:
The way I understand the documentation the combination of `START AT(...)` should be valid.